PhilipChat/src/chat.jsx

```diff
@@ -0,0 +1,72 @@
+import "./chat.css";
+import React, { useState, useEffect } from "react";
+
+export default function Chat({ author }) {
+  const [messages, setMessages] = useState([]);
+  const url = "https://chat.avalon.build/chat";
+
+  async function fetchMessages() {
+    const response = await fetch(url);
+    const messagesData = await response.json();
+    setMessages(messagesData);
+  }
+
+  useEffect(() => {
+    fetchMessages();
+  }, []);
+
+  const entirePage = messages.map((m) => {
+    if (author === m.author) {
+      return (
+        <div className="message author">
```

Comment:
illegal naming convention

Reply:
There is no issue here, it is just multiple classes.

Reply:
But you probably dont need multiple classes.